dev_guide/templates.adoc

```diff
@@ -105,13 +86,77 @@ $ osc process -f examples/sample-app/application-template-dockerbuild.json -v AD
 
 ====
 
-The JSON string can either be redirected to a file or applied directly by piping the process output to the `apply` command, as shown in the following example.
+The JSON string can either be redirected to a file or applied directly by piping the process output to the `create` command:
 
 .Redirecting JSON String
 ====
 
 ----
-$ osc process -f examples/sample-app/application-template-dockerbuild.json -v ADMIN_USERNAME=root,MYSQL_DATABASE=admin | osc apply -f -
+$ osc process -f examples/sample-app/application-template-dockerbuild.json -v ADMIN_USERNAME=root,MYSQL_DATABASE=admin | osc create -f -
+----
+
+====
+
+=== Creating from Template
+
+To create an configuration from a template using the Management Console:
+
+1. Click on the *Create+* button:
++
+====
+
+image:console_create.png["Console Create"]
+====
+
+2. Select a template from the list of templates in your project, or provided by OpenShift:
++
+====
+
+image:console_select_template.png["Select Template"]
+====
+
+3. Modify template parameters in the template creation screen:
++
+====
+
+image:create_from_template.png["Create from Template"]
+====
++
+<1> Template name and description.
+<2> Container images included in the template.
+<3> Labels to assign to all items included in the template. You can add and edit labels for resources.
+<4> Parameters defined by the template. You can edit values for parameters defined in the template here.
+
+== Labels
+link:../architecture/core_objects/kubernetes_model.html#label[Labels] are used to manage and organize generated resources, such as pods. The labels specified in the template are applied to every resource that is generated from the template.
+
+There is also the ability to add labels in the template from the command line.
+
+****
+`$ osc process -f _<filename>_ -l name=otherLabel`
+****
+
+== Parameters
+The list of parameters that you can override are listed in the `*parameters*` section of the template. Alternatively, you can override them with the following command, specifying the file to be used:
+
+****
+`$ osc process --parameters -f _<filename>_`
+****
+
+The following shows the output when listing the parameters for one of the *_openshift/origin/examples_* templates.
+
+.Example Template Output
```

Comment:
this also isn't template output, it's just showing the result of running osc process with the --parameters argument which just dumps the available parameters in the template, it does not actually process the template (so it cannot be piped to osc create)